.github/workflows/publish.yml

```diff
@@ -0,0 +1,64 @@
+name: Publish to npm on PR merge
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - main
+
+jobs:
+  publish:
+    if: github.event.pull_request.merged == true
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 22
+
+      - uses: pnpm/action-setup@v4
+        name: Install pnpm
+        with:
+          version: 9
+
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile --strict-peer-dependencies
+
+      - name: Install rimraf
+        run: pnpm i -g rimraf
+
```

Comment:
What does this do and is there a reason it wouldn't be part of the package.json instead?

Reply:
Fixed. `rimraf` is already installed in `package.json`